pybossa/api/project.py

```diff
@@ -66,6 +66,12 @@ def _preprocess_post_data(self, data):
             data["long_description"] = desc
         if not desc:
             data["description"] = description_from_long_description(desc, long_desc)
+        # set default data_access
+        if not data.get('info', {}).get("data_access"):
+            if not data.get("info"):
```

Comment:
Could this line be optimized as follows?

```python
if not 'info' in data:
```

Or perhaps to reduce both this line and the next one:

```python
data["info"] = data.get("info", {})
```